geometry_msgs/Vector3Stamped location
int32 id
string identity
time firstSeen
time lastSeen
